src/app/personside/infotabs/saksoversikt/SaksoversiktContainer.tsx

```diff
@@ -123,6 +126,12 @@ class SaksoversiktContainer extends React.Component<Props, State> {
         if (this.dokumentListeRef.current) {
             this.dokumentListeRef.current.focus();
         }
+
+        if (isViktigÅViteTema(sakstema)) {
```

Comment:
trenger du egentlig å legge denne i state..?

Reply:
ALT skal i state! :D I don't see why not

Reply:
Fra spøk til revolver, det decoupler jo Viktigåvite-komponenten helt fra det stedet hvor man velger sakstema. Som her bare er ~3 lag med å passe en callback, men andre steder er værre. Synes det er en fin vane